src/main/java/org/spongepowered/api/util/command/CommandSource.java

```diff
@@ -44,6 +46,17 @@
     String getName();
 
     /**
+     * Gets the locale used by this command source. If this
+     * {@link CommandSource} does have a {@link Locale} configured or does not
+     * support configuring a {@link Locale}, the system default is used.
+     *
+     * @return The locale used by this command source
+     */
+    default Locale getLocale() {
+        return Locale.getDefault();
```

Comment:
good point, another reason to add such a method here to avoid such missunderstandings.

However I'm not sure whether we should change it to US though.
* Because its an implementation detail
* The `Locale.getDefault()` is currently used in Common as well
  * [`SpongeText#getDefaultLocale()`](https://github.com/SpongePowered/SpongeCommon/blob/836d1d4a8cf8c735e47d51d57ed2e26b96af6d84/src/main/java/org/spongepowered/common/command/MinecraftCommandWrapper.java)
  * [`MinecraftCommandWrapper`](https://github.com/SpongePowered/SpongeCommon/blob/836d1d4a8cf8c735e47d51d57ed2e26b96af6d84/src/main/java/org/spongepowered/common/command/MinecraftCommandWrapper.java#L188)
* Its only a default method that can easily be overwriten.
* Our server should be stable enough to send proper/existing messages for any language (as long as at least one is provided)
  * That being said, if someone want to bundle the XY translations  with the server and set XY as his default locale, the server should use XY translations instead of the US ones.